crates/forge-runner/src/test_case_summary.rs

```diff
@@ -3,13 +3,46 @@ use crate::expected_result::{ExpectedPanicValue, ExpectedTestResult};
 use cairo_felt::Felt252;
 use cairo_lang_runner::short_string::as_cairo_short_string;
 use cairo_lang_runner::{RunResult, RunResultValue};
+use num_integer::Roots;
 use std::option::Option;
 
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Default)]
 pub struct GasStatistics {
     pub min: u128,
     pub max: u128,
+    pub mean: Option<u128>,
+    pub std_deviation: Option<u128>,
+}
+
+impl GasStatistics {
+    pub fn new(gas_usages: &[u128]) -> Self {
+        let max = gas_usages.iter().copied().reduce(u128::max).unwrap();
+        let min = gas_usages.iter().copied().reduce(u128::min).unwrap();
```

Comment:
Nit, I'd take `gas_usages` by value, so it's clear at the caller site that we need to clone this values